src/Serilog/LoggerConfiguration.cs

```diff
@@ -130,6 +130,9 @@ public ILogger CreateLogger()
                     disposable.Dispose();
             };
 
+            if (!_logEventSinks.Any())
```

Comment:
Can this can be moved prior to the dispose setup? e.g. ` Action dispose = () =>`